source/usage-examples/updateOne.txt

```diff
@@ -3,3 +3,56 @@ Update a Document
 =================
 
 .. default-domain:: mongodb
+
+You can update a document in a collection by using the ``UpdateOne()``
+method. 
+
+The following example passes a query filter and an update parameter to the
+``UpdateOne()`` method, which matches a document in the ``restaurants`` collection
+by the value of its ``_id`` field and creates a new field in that document called
+``avg_rating`` with a value of *4.4*:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/updateOne.go
+   :start-after: begin updateone
+   :end-before: end updateone
+   :emphasize-lines: 6
+   :language: go
+   :dedent:
+
+Click here <TODO: github link to file> to see a fully runnable example.
```

Comment:
Should this `TODO` be resolved before merging?

Reply:
No, we will revise the links before publishing!